libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.unique/thread.lock.unique.mod/release.pass.cpp

```diff
@@ -13,27 +13,25 @@
 // mutex_type* release() noexcept;
```

Comment:
We're not testing the `noexcept`. Please make a pass to ensure we're not missing it elsewhere.